modules/flight_input/flight_interface.py

```diff
@@ -0,0 +1,50 @@
+"""
+Creates flight controller and combines odometry data and timestamp.
+"""
+
+from modules.common.mavlink.modules import flight_controller
+from modules import odometry_and_time
+
+
+# This is just an interface
+# pylint: disable=too-few-public-methods
+class FlightInterface:
+    """
+    Create flight controller and combines odometry data and timestamp.
+    """
+    __create_key = object()
+
+    @classmethod
+    def create(cls, address: str) -> "tuple[bool, FlightInterface | None]":
+        """
+        address, TCP or UDP string.
+        """ 
+        result, controller = flight_controller.FlightController.create(address)
+
+        if result == False: 
+            return False, None
+
+        return True, FlightInterface(cls.__create_key, controller)
+
+    def __init__(self, class_private_create_key, controller):
+        """
+        Private constructor, use create() method.
+        """
+        if controller is None:
+            return False, None
```

Comment:
Remove, `controller` is guaranteed to not be None since it is guarded by the `create()` method.